packages/react-instantsearch/src/connectors/connectInfiniteHits.js

```diff
@@ -5,6 +5,10 @@ import {PropTypes} from 'react';
 
 import createConnector from '../core/createConnector';
 
+function getId() {
+  return 'pager';
```

Comment:
The id here is "public" API, it's available for url synchronisation via onStateChange() that's why I asked. Ultimately we might want to go for easy to understand IDS since now those are not directly synced in the URL (in another iteration)